docs/quickstarts/overview.mdx

```diff
@@ -26,7 +26,13 @@ description: Check out our Clerk getting started guides and tutorials.
 
   - [Astro](/docs/quickstarts/astro)
   - Easily add secure and SSR-friendly authentication to your Astro application with Clerk.
-  - {<svg viewBox="0 0 95 107"><path fill="url(#d59103f7)" d="M32.589 91.136c-4.834-4.418-6.245-13.703-4.231-20.429 3.492 4.24 8.33 5.584 13.342 6.343 7.737 1.17 15.335.732 22.523-2.804.822-.405 1.582-.944 2.48-1.489.675 1.957.85 3.932.615 5.942-.573 4.897-3.01 8.679-6.885 11.546-1.55 1.146-3.19 2.171-4.79 3.253-4.917 3.323-6.248 7.22-4.4 12.888.044.139.083.277.183.614-2.51-1.124-4.345-2.76-5.742-4.911-1.476-2.27-2.178-4.781-2.215-7.498-.018-1.323-.018-2.657-.196-3.96-.434-3.178-1.926-4.602-4.737-4.683-2.885-.085-5.166 1.699-5.772 4.507-.046.216-.113.428-.18.679l.005.002Z"/><path style={{ fill: 'var(--light, #17191E) var(--dark, white)' }} d="M5 69.587s14.314-6.973 28.668-6.973L44.49 29.12c.405-1.62 1.588-2.72 2.924-2.72 1.335 0 2.518 1.1 2.924 2.72L61.16 62.614c17 0 28.668 6.973 28.668 6.973S65.514 3.352 65.467 3.219C64.769 1.261 63.591 0 62.003 0H32.827c-1.588 0-2.718 1.261-3.464 3.22C29.311 3.35 5 69.586 5 69.586Z"/><defs><linearGradient id="d59103f7" x1="27.47" x2="74.145" y1="107" y2="84.947" gradientUnits="userSpaceOnUse"><stop stopColor="#D83333"/><stop offset="1" stopColor="#F041FF"/></linearGradient></defs></svg>}
+  - {<svg viewBox="0 0 36 36" style={{ fill: 'var(--light, #121212) var(--dark, white)' }}><path d="M13.2807 30.6633C11.654 29.1784 11.1793 26.0511 11.8565 23.79C13.0332 25.2164 14.6598 25.6686 16.345 25.9229C18.9481 26.3166 21.5062 26.1704 23.9225 24.9802C24.1992 24.8452 24.4557 24.6652 24.7572 24.4807C24.9844 25.1377 25.0429 25.8014 24.9642 26.4786C24.7707 28.1255 23.9518 29.3989 22.6468 30.3619C22.1271 30.7488 21.5759 31.0931 21.0359 31.4575C19.3823 32.5757 18.9346 33.8874 19.5555 35.793L19.6185 36C18.8243 35.6542 18.1514 35.0792 17.6859 34.3486C17.1994 33.5966 16.9408 32.7199 16.9412 31.8243C16.9344 31.381 16.9344 30.9311 16.8737 30.4923C16.7297 29.4237 16.228 28.9444 15.2808 28.9174C14.8281 28.891 14.3813 29.0304 14.024 29.3095C13.6666 29.5886 13.4232 29.9883 13.3392 30.4339C13.3234 30.5058 13.3009 30.5778 13.2784 30.6611H13.2807V30.6633ZM4 23.412C4 23.412 8.8147 21.0677 13.6451 21.0677L17.2854 9.79814C17.4204 9.25367 17.8209 8.8847 18.2686 8.8847C18.7186 8.8847 19.1168 9.25367 19.254 9.79814L22.8943 21.0677C28.6135 21.0677 32.5395 23.4143 32.5395 23.4143L24.3432 1.08218C24.1092 0.425224 23.7133 0 23.1778 0H13.3617C12.8284 0 12.4482 0.425224 12.1962 1.08218L4 23.4143V23.412Z"/></svg>}
```

Comment:
Please can we revert the change to the Astro logo here?

```suggestion
  - {<svg viewBox="0 0 95 107"><path fill="url(#d59103f7)" d="M32.589 91.136c-4.834-4.418-6.245-13.703-4.231-20.429 3.492 4.24 8.33 5.584 13.342 6.343 7.737 1.17 15.335.732 22.523-2.804.822-.405 1.582-.944 2.48-1.489.675 1.957.85 3.932.615 5.942-.573 4.897-3.01 8.679-6.885 11.546-1.55 1.146-3.19 2.171-4.79 3.253-4.917 3.323-6.248 7.22-4.4 12.888.044.139.083.277.183.614-2.51-1.124-4.345-2.76-5.742-4.911-1.476-2.27-2.178-4.781-2.215-7.498-.018-1.323-.018-2.657-.196-3.96-.434-3.178-1.926-4.602-4.737-4.683-2.885-.085-5.166 1.699-5.772 4.507-.046.216-.113.428-.18.679l.005.002Z"/><path style={{ fill: 'var(--light, #17191E) var(--dark, white)' }} d="M5 69.587s14.314-6.973 28.668-6.973L44.49 29.12c.405-1.62 1.588-2.72 2.924-2.72 1.335 0 2.518 1.1 2.924 2.72L61.16 62.614c17 0 28.668 6.973 28.668 6.973S65.514 3.352 65.467 3.219C64.769 1.261 63.591 0 62.003 0H32.827c-1.588 0-2.718 1.261-3.464 3.22C29.311 3.35 5 69.586 5 69.586Z"/><defs><linearGradient id="d59103f7" x1="27.47" x2="74.145" y1="107" y2="84.947" gradientUnits="userSpaceOnUse"><stop stopColor="#D83333"/><stop offset="1" stopColor="#F041FF"/></linearGradient></defs></svg>}
```

Reply:
Thank you, seems that I messed up the rebase